core2/src/main/kotlin/motif/core/ScopeGraph.kt

```diff
@@ -19,7 +19,7 @@ import motif.ast.IrType
 import motif.models.ChildMethod
 import motif.models.Scope
 
-class Child(val parent: Scope, val method: ChildMethod, val scope: Scope)
+class ScopeEdge(val parent: Scope, val method: ChildMethod, val scope: Scope)
```

Comment:
Change param ordering and naming to: `(val parent: Scope, val child: Scope, val method: ChildMethod)`